integration/mock.graphql

```diff
@@ -41,3 +43,6 @@ type Task {
 type Query {
   helloWorld: String
 }
+
+scalar GraphbackTimestamp
```

Comment:
The best benefit of having embedded scalars is to not have to define them in model. What this means that our schema plugin need to add them etc. WDYT?

Reply:
> The best benefit of having embedded scalars is to not have to define them in model. 

+1 on this. I actually started with this in mind, till I ran through issues "unknown GraphbackTimestamp type" when running my tests. so adding them to the schema was the solution. 

Happy to chat about it if there are suggestion as that would be of huge benefit.

Reply:
Yes.. It will mean that model files will not be valid GraphQL schemas but we can always append all scalar definitions to them so things will work

Reply:
This would be a breaking change as the Graphback API allows a GraphQLSchema object or a string as the model. It would also mean a change to the templates to load the schema as a string and not with `loadSchemaSync`.

Should this happen in a separate PR? Nothing from current PR would need to be changed as such for this to happen, but the scope is quite different and it would have an effect in other areas of the library.

Reply:
@craicoverflow thanks for the analysis. Maybe we get this PR in first with proper documentation and evaluate further?

Reply:
Let's drop new issue and cover it by triaging. The way we can do it is to generate model file initially that contains those scalars and people can manage them etc.

Reply:
https://github.com/aerogear/graphback/issues/1756